docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+# Creating a Bridge With a Dead Letter Sink (DLS)
+
+## What is a Dead Letter Sink?
+
+Knative Eventing provides various configuration parameters to control the delivery of events in case of failure. For instance, you can decide to retry sending events that failed to be consumed, and if this didn't work you can decide to forward those events to a dead letter sink.
+
```

Comment:
Also, can the dead letter sink be anything (e.g. a TriggerMesh function, a TriggerMesh Target, ...)? or does it have to be a Knative service?

Reply:
well.. both triggermesh function's and target's are actually knative services.

Reply:
so im not posive what the question is really. are you asking if it can be linked to non-knative services/deployments?

Reply:
if so... im not posive. I would have to experiment or phone a friend lol